Packs/Oletools/Scripts/OletoolsScript/OletoolsScript.py

```diff
@@ -0,0 +1,211 @@
+from CommonServerPython import *
+import demistomock as demisto
+import oletools.oleid
+from oletools.olevba import VBA_Parser
+import subprocess
+from oletools import crypto
+import os
+import hashlib
+
+
+class OleClient:
+    def __init__(self, file_info, ole_command, password=None, decoded=False):
+        self.name = file_info['name']
+        self.file_path = file_info['path']
+        self.password = password
+        self.show_decoded = decoded
+        self.decrypted_file_path = None
+        self.processed_file_path = file_info['path']
+        self.ole_command = ole_command
+        self.hash = None
+
+    def __del__(self):
+        try:
+            if self.password and self.decrypted_file_path:
+                os.unlink(self.decrypted_file_path)
+        except Exception:  # e.g. file does not exist or is None
+            pass
+
+    def decryption(self):
+        if crypto.is_encrypted(self.file_path) and self.password:
+            try:
+                passwords = [self.password] + crypto.DEFAULT_PASSWORDS
+                self.decrypted_file_path = crypto.decrypt(self.file_path, passwords)
+                if not self.decrypted_file_path:
+                    raise crypto.WrongEncryptionPassword(self.file_path)
+            except Exception as e:
+                raise DemistoException(f'The file decryption failed with the following message:\n {e}')
+
+    @staticmethod
+    def calc_hash(file_path: str):
+        with open(file_path, "rb") as f:
+            b = f.read()  # read entire file as bytes
+            return hashlib.sha256(b).hexdigest()
+
+    def run(self):
+
+        self.decryption()
+
+        if self.decrypted_file_path:
+            self.processed_file_path = self.decrypted_file_path
+
+        # calculate the file hash
+        self.hash = self.calc_hash(self.processed_file_path)
+
+        if self.ole_command == 'oleid':
+            cr = self.oleid()
+        elif self.ole_command == 'oleobj':
+            cr = self.oleobj()
+        elif self.ole_command == 'olevba':
+            cr = self.olevba()
+        else:
+            raise NotImplementedError('Command "{}" is not implemented.'.format(self.ole_command))
+
+        self.wrap_command_result(cr)
+        return cr
+
+    def wrap_command_result(self, cr: CommandResults):
+        cr.outputs = {'sha256': self.hash, 'file name': self.name, 'ole_command_result': cr.outputs}
```

Comment:
As discussed, please change "file name" to "file_name".
Also, please verify there are no other outputs with space character.